### ISSUE TITLE
Add "Boardwalk" as a type of Structure

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#344]
Moving over from here https://github.com/nationalparkservice/places-editor/issues/94.

This has not been added to the preset spreadsheet. cc @mtaylorlong

**Comment by chadlawlis** [comment]
FYI @asouthwould

**Comment by mtaylorlong** [comment]
Looks like adding a surface of 'boardwalk' to my YELL data is successfully dumping this into our mapbox tiles as an attribute. So, it's working!

I'll keep this open as a reminder to recommend adding it to the presets spreadsheet next time that's up for review.

**Comment by chadlawlis** [comment]
Presets have been regenerated and this was not added to the spreadsheet.

If you'd like to see this added feel free to include it in the spreadsheet, @mtaylorlong.

**Comment by mtaylorlong** [comment]
I'm actually okay waiting on this. I think this discussion would go well with discussing the addition of tunnels and bridges as line presets.

**Comment by nateirwin** [comment]
Up to this point, we've used the "structures" radio buttons in Places Editor to tag segments of roads and trails as "bridge", "crosswalk", "tunnel", etc. Do we need to re-think how we do this?

**Comment by mtaylorlong** [comment]
If that's how it's done in OSM, I think that makes sense. Adding these as a preset just helps editors  by allowing them to search for and apply the tag without having to do so manually.

If the structures tag is already being used in some areas, my next request would be that it gets added into the attributes for the roads_and_trails vector tiles.

**Comment by nateirwin** [comment]
Yeah, it's being used quite a bit. @jimmyrocks just needs to get it added.

**Comment by mtaylorlong** [comment]
Update: I just checked in with @jimmyrocks on this one and we're already rendering this structure info out to mapbox vector tiles. So scratch what I said earlier. Bridges, Tunnels, etc. are already handled – I just need to add a style to accommodate them (which I will test in YELL).

One could argue that it's difficult to distinguish between a bridge for a trail and a boardwalk. A boardwalk could be seen as a kind of bridge. However, in YELL, I use the surface tag to assign something as a boardwalk. This means that I can still distinguish where the boardwalk actually becomes a bridge, since they're separate concepts.

So, my main request here was that we make Boardwalk a proper searchable preset. At the miminum, it should be part of the drop down options under surface (if my approach makes sense), but it would be even better as a full-on preset. Then people might actually use it.

**Comment by nateirwin** [comment]
I guess the reason I prefer adding these "structure" or "material" attributes as tags on other presets and not presets themselves is because our presets are currently organized around _usage_. So if we add a `boardwalk` preset, how do we know if the boardwalk is part of a larger equestrian trail?

To me, `boardwalk` is really just about the material and we shouldn't add it as a standalone preset. This seems to be consistent with what we've done up to this point.

**Comment by mtaylorlong** [comment]
@nateirwin: That actually makes a lot of sense. I'm at peace with keeping Boardwalk as a surface tag. I'll dial back my request to ask only that it be added to the drop-down of options under that setting.

**Comment by mtaylorlong** [comment]
I just renamed this issue since we're taking it in a different direction. After revisiting this issue, it still makes sense to classify boardwalks as a type of Structure. They really do fall into a similar category as a bridge or tunnel – whatever type of trail runs over a boardwalk, that type should be maintained separately.

So, I'm proposing we identify Boardwalks with the tag `bridge=boardwalk` and add it to Editor as an option under Structure. When we write this data out to our tiles, Boardwalks should have a structure attribute of `boardwalk` (lower case).

@jimmyrocks, @cwlawlis802: Does that sound good to y'all?
